src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -177,3 +177,6 @@
 CONST_PERISCOPE_RELEASE_TAG = "v0.9"
 CONST_PERISCOPE_IMAGE_VERSION = "0.0.9"
 CONST_PERISCOPE_NAMESPACE = "aks-periscope"
+
+CONST_KEYVAULT_NETWORK_ACCESS_PUBLIC = "Public"
```

Comment:
to be consist, KEYVAULT -> AZURE_KEYVAULT?

Reply:
renamed